dev-packages/application-manager/src/generator/frontend-generator.ts

```diff
@@ -380,12 +380,16 @@ app.on('ready', () => {
         // We want to pass flags passed to the Electron app to the backend process.
         // Quirk: When developing from sources, we execute Electron as \`electron.exe electron-main.js ...args\`, but when bundled,
         // the command looks like \`bundled-application.exe ...args\`.
-        const cp = fork(mainPath, process.argv.slice(devMode ? 2 : 1), { env: Object.assign({
+        const cp = fork(mainPath, [], { env: Object.assign({
```

Comment:
I think this change means that no Theia command line arguments will be picked up. E.g. `--log-level` and `--plugins`. However, it looks like this would have been true for the 1.2.1 release that was based on Theia 1.2, since the arguments were never passed to the forked process?

When we update to Theia 1.5, we should consider upstreaming a fix for the new `ElectronMainApplication` class. If I understand the problem correctly, it should be possible to update the `getProcessArgvWithoutBin` to handle the additional "Helper" argument present on Mac:
https://github.com/eclipse-theia/theia/blob/master/packages/core/src/electron-main/electron-main-application.ts#L190-L194

Reply:
I've checked, and I was correct that in 1.2.1 the command line arguments were not passed to the forked process when the app is packaged, so this change isn't a regression.

Also after this change, we need to set `--no-cluster` to get arguments like `--hosted-plugin-inspect` to apply in dev mode, but that is currently set in our vscode launch configurations.